backend/api/Areas/Project/Controllers/DisposeController.cs

```diff
@@ -143,8 +143,8 @@ public async Task<IActionResult> UpdateProjectAsync(string projectNumber, Projec
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "To support standardized routes (/{projectNumber})")]
         public async Task<IActionResult> DeleteProjectAsync(string projectNumber, ProjectModel model)
         {
-            await _pimsService.Project.RemoveAsync(_mapper.Map<Entity.Project>(model));
-            return new JsonResult(model);
+            var project = await _pimsService.Project.RemoveAsync(_mapper.Map<Entity.Project>(model));
+            return new JsonResult(_mapper.Map<ProjectModel>(project));
```

Comment:
The delete project was an interesting case. Didn't make sense to include all properties on a project within the project list view, which would provide the required metadata to know a subdivision exists on a project.

So instead what I went with was updating the delete api to return the properties. This allows the FE to display a warning after the delete occurs, which I think is fine in this case.